src/elm/Page/Login.elm

```diff
@@ -328,3 +382,33 @@ renderAuthInfo model =
 
             else
                 text ""
+
+
+additionalInfos : AuthType -> Html Msg
+additionalInfos loginType =
+    case loginType of
+        Token ->
+            Form.row [ Row.attrs [ Spacing.mt5 ] ]
+                [ Form.col [ Col.sm12 ]
+                    [ Grid.containerFluid
+                        [ Border.all
+                        , Border.rounded
+                        , Spacing.p2
+                        , class "bg-light"
+                        ]
+                        [ Html.text "To generate a token from a realm key, you can use "
```

Comment:
A valid JWT token should be used, you can use <a href="....">astartectl</a> to generate one: